Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/ui/Question/QuestionAddServlet.java

```diff
@@ -0,0 +1,58 @@
+package ui.Question;
+
+import entity.Quiz;
+import entity.Question;
+import org.apache.log4j.Logger;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.*;
+
+
+public class QuestionAddServlet extends HttpServlet{
+    private static Logger log = Logger.getLogger(QuestionAddServlet.class.getName());
+
+
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        this.process(request, response);
+    }
+
+
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        this.process(request, response);
+    }
+
+
+    private void process(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        response.getWriter().print("Some text");
```

Comment:
Some text?

Reply:
This was needed in the initial stages to test the servlet operation. Forgot to remove in future versions